hazelcast/src/main/java/com/hazelcast/internal/management/ManagementCenterService.java

```diff
@@ -637,8 +640,7 @@ public void memberAdded(MembershipEvent membershipEvent) {
             try {
                 Member member = membershipEvent.getMember();
                 if (member != null && instance.node.isMaster() && urlChanged) {
-                    Operation operation = new UpdateManagementCenterUrlOperation(managementCenterUrl);
-                    callOnMember(member, operation);
+                    resolveFuture(callOnMember(member, new UpdateManagementCenterUrlOperation(managementCenterUrl)));
```

Comment:
I 'd rather keep the `Operation` construction on a separate line for readability.